agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -41,6 +42,7 @@ class AppVariantDB(Model):
     user_id: UserDB = Reference(key_name="user")
     parameters: Dict[str, Any] = Field(default=dict)
     previous_variant_name: Optional[str]
+    organisation: Optional[OrganizationDB]
```

Comment:
why is it optional?

Reply:
or maybe that is good, this way we can find a way not to migrate. But just update these on run

Reply:
Yes, it should be optional for now until we've successfully migrated everything.